src/output.cpp

```diff
@@ -174,7 +174,7 @@ void trim_and_print(WINDOW *w, int begin_y, int begin_x, int width, nc_color bas
             iLength += iTempLen;
 
             if ( iLength > width ) {
-                sTempText = sTempText.substr(0, cursorx_to_position(sTempText.c_str(), iTempLen - (iLength - width) - 1, NULL, -1)) + "…";
+                sTempText = sTempText.substr(0, cursorx_to_position(sTempText.c_str(), iTempLen - (iLength - width) - 1, NULL, -1)) + "...";
```

Comment:
No problem. I understand why you change that but I disagree with solution. Could you try to use "\u2026" sequence (with comment what is it).

BTW, this solution is used in function ```utf8_wrapper::shorter()``` in ```src\catacharset.cpp``` (line ~602, close to the end of file), so you can use it as example, or even modify functions for deduplication.